src/ziggurat/init.clj

```diff
@@ -187,3 +191,6 @@
        (log/error e)
        (stop stop-fn modes)
        (System/exit 1)))))
+
+(defn -init [args]
+  (main (util/java->clojure-map args)))
```

Comment:
java-map->clojure-map?

Reply:
a java map is passed to the function, we convert it to a clojure map

Reply:
Function name

Reply:
done